src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'StyleLint';
+
+export default async function stylelintCheck() {
```

Comment:
one thing to consider for this check is to make sure that the Stylelint config matches the config that we made

since [our template](https://github.com/upleveled/eslint-config-upleveled/blob/main/templates/stylelint.config.cjs) is pretty complicated, you may want to publish a new package as mentioned here:

- https://github.com/upleveled/eslint-config-upleveled/issues/165#:~:text=package%20for%20Stylelint%20config

Reply:
[e8323b9](https://github.com/upleveled/preflight/pull/377/commits/e8323b99bd94040a482eb9c08dc7f848d2664911)